HotelReservation/HotelReservation/ReservationSteps/Reservation/ReservationStartProcess.cs

```diff
@@ -4,13 +4,25 @@ namespace HotelReservation.ReservationSteps.Reservation
 {
     public class ReservationStartProcess : IReservationStep
     {
-        string name;
```

Comment:
Wydaje mi się, że w dalszym ciągu nie wymyśliłeś, jak chcesz używać inputów w stepie, a zacząłeś implementować ich obsługę wyżej w kodzie. Na przykład przekazujesz z samej góry typeof(string), co nie ma na nic wypływu. 

Najłatwiej, jak to sobie przegadamy w 4 oczy :) Ustawmy się w poniedziałek

Reply:
Fixed